.gitignore

```diff
@@ -27,3 +27,4 @@ contrib/examples/vendor/*
 integration.test*
 debug.test*
 .cache
+/api_violations.txt
```

Comment:
Where does this new file come from?

Reply:
As I'm sure you later found out with the review, looks to be generated by the openapi_gen script invoked from build/update-apiserver-gen.sh.  Looks pretty standard.

Reply:
I was not sure where to put that file. Yes, it is a generated file that is only temporarily needed.